### PR TITLE
Added ProjectEstimation model

### PR DESCRIPTION
**Pull request by maxceem**
Implementation for https://github.com/topcoder-platform/tc-project-service/issues/305

This PR has been tested locally together with the corresponding changes to Connect App and works good together https://github.com/appirio-tech/connect-app/pull/3058

Migration SQL script has to be run.

### CODE REVIEW

File: migrations/20190526_project_estimation.sql [hunk 1]
```diff
@@ -0,0 +1,35 @@
+--
+-- CREATE NEW TABLE:
+--   project_estimations
+--
+
+CREATE TABLE project_estimations
+(
+    id bigint NOT NULL,
+    "buildingBlockKey" character varying(255) NOT NULL,
+    conditions character varying(255) NOT NULL,
```
Comment (vikasrohit) [inline migrations/20190526_project_estimation.sql]: we can make it to `512` for future.

Comment (maxceem) [reply migrations/20190526_project_estimation.sql]: @vikasrohit if you didn't run migration script yet, I can update it to `512` now.

Comment (maxceem) [reply migrations/20190526_project_estimation.sql]: Pushed commit where set it to `512`. https://github.com/topcoder-platform/tc-project-service/pull/306/commits/495f65e3f94e4c23a229ec29d6fd8d3480475e74

Comment (vikasrohit) [reply migrations/20190526_project_estimation.sql]: Thanks

**Pull request merged**